egs2/librimix/enh1/conf/tuning/train_enh_codecformer.yaml

```diff
@@ -0,0 +1,56 @@
+optim: adam
+init: xavier_uniform
+max_epoch: 1
+batch_type: folded
+batch_size: 6 # batch_size 16 can be trained on 4 RTX 2080ti
+iterator_type: chunk
+chunk_length: 24000
+num_workers: 4
+optim_conf:
+    lr: 0.00015
+    weight_decay: 0
+patience: 5
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - si_snr
+    - max
+-   - valid
+    - loss
+    - min
+keep_nbest_models: 1
+scheduler: reducelronplateau
+scheduler_conf:
+    mode: min
+    factor: 0.5
+    patience: 1
+model_conf:
+    loss_type: si_snr
```

Comment:
This argument has been deprecated. Please follow https://github.com/espnet/espnet/blob/master/egs2/wsj0_2mix/enh1/conf/tuning/train_enh_conv_tasnet.yaml#L52-L60 to set the criterion.